### PR TITLE
Added update_vocs to snakefile

### PR DESCRIPTION
**Pull request by favelava**
Adds update_vocs to snakemake file

### CODE REVIEW

File: workflow_main/Snakefile [hunk 1]
```diff
@@ -383,4 +386,12 @@ rule surveillance_data:
...
-        """
\ No newline at end of file
+        """
+
+rule update_vocs:
+    output:
+        voc_list = os.path.join(config["static_data_folder"], "vocs.json")
+    shell:
+        """
+        python3 scripts/update_vocs -o {output.voc_list}
```
Comment (atc3) [inline workflow_main/Snakefile:396]: Shouldn't this be `update_vocs.py`?

**Review by atc3 — COMMENTED** [review]
Not sure if python will find the file without the exact matching name (see the comment), did you test this?

**Comment by favelava** [comment]
I must have push an outdated version of the branch and not caught it. I retested the snakemake file and it looks like everything is set now.

**Review by atc3 — APPROVED** [review]
LGTM

**Pull request merged**